test/nimble.phx.gen.template/release_version_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Nimble.Phx.Gen.Template.ReleaseVersionTest do
+  use ExUnit.Case, async: true
+
+  alias Nimble.Phx.Gen.Template.Hex.Package
+
+  @tag :release_version
+  test "the new version should be greater than the hex version" do
```

Comment:
Prefer the imperative tone in tests 🙈 

```suggestion
  test "the new version is greater than the hex version" do
```

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/71/commits/c0e6ac5f2c7d453c82d1b1073354c10dba1e378f @olivierobert ✅